tests/platform_tests/api/test_thermal.py

```diff
@@ -0,0 +1,175 @@
+import logging
+import random
+import re
+import time
+
+import pytest
+import yaml
+
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.helpers.platform_api import chassis, thermal
+
+from platform_api_test_base import PlatformApiTestBase
+
+logger = logging.getLogger(__name__)
+
+pytestmark = [
+    pytest.mark.sanity_check(skip_sanity=True),
```

Comment:
Remove this line. We're no longer allowing any tests to skip the sanity check.

Reply:
fixed